policyengine_us/variables/gov/states/or/tax/income/liheap/or_liheap_heat_in_rent.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class heating_costs_included_in_rent(Variable):
```

Comment:
```suggestion
class or_heat_in_rent(Variable):
```
Make sure this name is the same as the file's name.